examples/programmatic/tensor_basis/tensor_basis.py

```diff
@@ -0,0 +1,191 @@
+# ``model.pt`` and the ``extensions`` directory containing some required files for the
+# model to be used. In the ``outputs`` directory, under nested directories named after
+# the date and time of the beginning of the training, you will find the training logs
+# and model checkpoints.
+
+# %%
+#
+# In case you need to export a specific checkpoints, you can do so using:
+#
+# .. code:: bash
+#
+#    mtt export /path/to/model.ckpt -o model_name.pt
+#
+
+# %%
+#
+# To evaluate the model, we can write the following ``eval.yaml`` file:
+#
+# .. code:: yaml
+#
+#    systems:
+#        read_from: bulk_water_100.xyz
+#        length_unit: angstrom
+#
+
+# %%
+#
+# And then evaluate the model using the command:
+#
+# .. code:: bash
+#
+#    mtt eval model.pt eval.yaml -e extensions/ -o outputs.mts 2> err.log
+#
+# The ``stderr`` output will be redirected to the ``err.log`` file to avoid seeing too
+# many warnings.
+
+# %%
+#
+# After evaluation is completed, three files will be created in the current directory.
+# The only one relevant for this example is ``outputs_mtt::polarizability.npz``,
+# containing the predicted values of the target tensorial properties as binary
+# ``TensorMap`` objects.
+
+# %%
+#
+# Load the predicted values of the target tensorial properties and the
+# training/validation/test indices to slice the predicted values. Since the indices are
+# inside the outputs directory, we need to find the latest folder containing the
+# indices.
+# Since we passed the scalar component of the polarizability as an "energy" target, we
+# after loading the predicted values we can update the metadata to reflect the correct
+# information about the target.
+
+predicted_polarizabilities = mts.load("outputs_mtt::polarizability.npz")
+
+index_folder = sorted(glob("outputs/*/*/indices"))[-1]
+indices = {
+    "train": np.loadtxt(f"{index_folder}/training.txt", dtype=int),
+    "val": np.loadtxt(f"{index_folder}/validation.txt", dtype=int),
+    "test": np.loadtxt(f"{index_folder}/test.txt", dtype=int),
+}
+
+# %%
+#
+# Plot the parity plots of the predicted values of the target tensorial properties
+# against the true values for the training, validation, and test sets.
+
+fig, axes = plt.subplots(1, 2, figsize=(10, 5))
+
+for key, ax in zip(spherical_tensormap.keys, axes):
+    o3_lambda = key["o3_lambda"]
+    o3_sigma = key["o3_sigma"]
+    ax.set_aspect("equal")
+    ax.set_xlabel("Target")
+    ax.set_ylabel("Prediction")
+    ax.set_title(f"$\lambda={o3_lambda}$, $\sigma={o3_sigma}$")
+    target = spherical_tensormap[key].values
+    prediction = predicted_polarizabilities[key].values
+    ax.set_xlim(target.min(), target.max())
+    ax.set_ylim(target.min(), target.max())
+    ax.plot([target.min(), target.max()], [target.min(), target.max()], "--k")
+
+    for tset, idx in indices.items():
+        ax.plot(target[idx].flatten(), prediction[idx].flatten(), ".", label=tset)
+    ax.legend()
+fig.tight_layout()
+plt.show()
```

Comment:
Some spurious lines seem to be printed in the docs here. It would be good to eliminate them, but I'm not sure how. Perhaps we can take inspiration from the other examples and tutorials